diff --git a/Release/makefile b/Release/makefile
index 5e75fc4..b6e3288 100644
--- a/Release/makefile
+++ b/Release/makefile
@@ -44,7 +44,7 @@ all: supla-dev
 supla-dev: $(OBJS) $(USER_OBJS)
 	@echo 'Building target: $@'
 	@echo 'Invoking: Cross G++ Linker'
-	g++ -pthread -lrt -o "supla-dev" $(OBJS) $(USER_OBJS) $(LIBS)
+	$(CXX) $(CXXFLAGS) -pthread -lrt -o "supla-dev" $(addprefix -o ,$(OBJS)) $(USER_OBJS) $(LIBS)
 	@echo 'Finished building target: $@'
 	@echo ' '
 
