diff --git a/Release/makefile b/Release/makefile
index a0e4742..bf231fe 100644
--- a/Release/makefile
+++ b/Release/makefile
@@ -45,7 +45,7 @@ all: supla-server
 supla-server: $(OBJS) $(USER_OBJS)
 	@echo 'Building target: $@'
 	@echo 'Invoking: Cross G++ Linker'
-	g++ -L/usr/include/mysql -L/usr/src/openssl/lib -pthread -lrt -o "supla-server" $(OBJS) $(USER_OBJS) $(LIBS)
+	g++ -L/usr/lib64/mysql -L/usr/include/mysql -L/usr/src/openssl/lib -pthread -lrt -o "supla-server" $(OBJS) $(USER_OBJS) $(LIBS)
 	@echo 'Finished building target: $@'
 	@echo ' '
 
